src/store/collectibles/reducers.ts

```diff
@@ -1,12 +1,13 @@
 import { getType } from 'typesafe-actions';
 
-import { Collectible, CollectiblesState } from '../../util/types';
+import { AllCollectiblesFetchStatus, Collectible, CollectiblesState } from '../../util/types';
 import * as actions from '../actions';
 import { RootAction } from '../reducers';
 
 const initialCollectibles: CollectiblesState = {
     collectibleSelected: null,
     allCollectibles: {},
+    allCollectiblesFetchStatus: AllCollectiblesFetchStatus.Request,
```

Comment:
Since we perform periodical refresh/updates, I thought about setting this as `Request` only on the initial state and show the loading/spinner in that moment.